litmus-portal/backend/self-deployer/template/self-deployer.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
How this manifest different from self-deployer-manifest.yml?

Reply:
This is used by client-go to just delete the self-deployer and not the roles. We can reuse the roles for other purpose that's why it is here.

Reply:
We can put the cluster-admin in k8s manifest and  then you can reuse that self-deployer-manifest.yml. 
And where is subscriber manifest(agent + igc)?

Reply:
The manifest for the igc+agent is present in the graphql-server/template which is in the other [PR](https://github.com/litmuschaos/litmus/pull/1771). Yes we if we shift the roles we can maybe use it but building the docker image might become a little difficult as we'll have to build the image from the litmus-portal dir as we cannot access files outside the context dir and if we change the context dir, it might increase the context size and slow down the build process.